collector/runtime-benchmarks/asynctest/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "asynctest"
```

Comment:
Eventually, this should be ideally called e.g. `async`, rather than `asynctest`.